rt/transports/websocket/src/main/java/org/apache/cxf/transport/websocket/jetty12/Jetty12WebSocketDestination.java

```diff
@@ -335,5 +342,36 @@ private HttpServletRequest getHttpServletRequest(final UpgradeRequest upgradeReq
         }
     }
 
+    @org.eclipse.jetty.websocket.api.annotations.WebSocket
+    public class Jetty12WebSocket {
```

Comment:
Do we need this class to be public?

```suggestion
   static class Jetty12WebSocket {
```

Reply:
Yes, this needs to be public, so that the annotation of this class
    @org.eclipse.jetty.websocket.api.annotations.WebSocket
can take effect